test/metabase/driver/sql/parameters/substitution_test.clj

```diff
@@ -93,7 +93,7 @@
   (mt/with-clock #t "2018-07-01T12:30:00.000Z"
     (mt/with-metadata-provider meta/metadata-provider
       (let [field-filter (params/map->FieldFilter
-                          {:field (lib.metadata/field (qp.store/metadata-provider) (meta/id :checkins :date))
+                          {:field (lib.metadata/field (qp.store/metadata-provider) (meta/id :orders :created-at))
```

Comment:
It's unexpected for me that the rest of the test doesn't need changing. Where does/should `"PUBLIC"."CHECKINS"` come from?